tinker/static/assets/js/tinker.js

```diff
@@ -14,4 +14,117 @@ $(document).ready(function () {
             $(this).text("- Less Search Filters")
         }
     });
-});
\ No newline at end of file
+});
+
+function pagination() {
+    var numberOfItems = $(" #loop .list-group").length;
+    var limitPerPage = $("#first").val();
+
+    $("#selected-option").change(function() {
+        var o = $(this).find("option:selected");
+        if (o.val() === "All") {
+            limitPerPage = numberOfItems;
+        } else {
+            limitPerPage = o.val()
+        }
+
+        $(".pagination").children("li.to-remove").remove();
+
+        var totalPages = Math.ceil(numberOfItems / limitPerPage);
+
+        createButtons(totalPages, limitPerPage);
+
+        var currentPage = 1;
```

Comment:
is currentPage always set to 1?

Reply:
Whenever you change how many items you want per page, it resets the current page to 1. So if we have 100 items and 10 times per page, then we have 10 pages, but if we set it to show all on one page, then the current page can't be the same as whatever page you were on previous (if it was greater than 1) so I just set current page to 1.

Reply:
That makes sense. Maybe add a comment here?

Reply:
Added a comment